minimint-api/src/lib.rs

```diff
@@ -104,6 +113,14 @@ impl Amount {
         }
     }
 
+    pub fn from_str_in(s: &str, denom: Denomination) -> Result<Amount, ParseAmountError> {
+        if let Denomination::MilliSatoshi = denom {
+            return Self::from_str(s);
+        }
+        let btc_amt = bitcoin::util::amount::Amount::from_str_in(s, denom)?;
```

Comment:
Ok yes, to fix that we would need to implement something like bitcoins `from_str_in` and this could be alot of code just to be able to use `0.1sat` which is not conventional in the first place imo.